### PR TITLE
This adds preconfigured tests and CI to the template

### PR DESCRIPTION
**Pull request by kayibal**
# Whats new
1. default circleci configuration
2. Example unit test
3. Example integration test

# How is it tested
It depends on a specific drtools branch: feature/clean_root_dir. Thus 1) is untested. 2) & 3) are tested by instantiating the template locally and running commands from the docs.

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/README.md [hunk 1]
```diff
@@ -73,6 +73,44 @@ To run without dev mode it is sufficient to specify the `-f docker-compose.yml`
...
+# Build image tagged as current branch name
+docker build- t drtools/{{cookiecutter.repo_name}}:$TAG .
+```
+
+## Tests
+Unittests usually run on your local machine and don't depend on any services.
+
+During integration tests your local machine serves as controller. This usually
+makes debugging easier. Services must be started with docker-compose. This 
+requires you though to have the current project installed in your python env:
```
Comment (michcio1234) [inline {{ cookiecutter.repo_name }}/README.md]: Either missing code block or unnecessary colon at the end of last sentence. Or maybe "Integration tests" section should be level 3 (###), not level 2.

File: {{ cookiecutter.repo_name }}/.circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: drtools/drtools:latest
```
Comment (michcio1234) [inline {{ cookiecutter.repo_name }}/.circleci/config.yml:5]: Why this image? I've seen that in is24 we're using `docker:17.05.0-ce-git`. What's the advantage of using our image here?

**Review by michcio1234 — COMMENTED** [review]
Okay for me, I just have one question and one minor remark. Feel free to merge after changing this one small thing.

File: {{ cookiecutter.repo_name }}/.circleci/config.yml (hunk 2 above)
Comment (kayibal) [reply {{ cookiecutter.repo_name }}/.circleci/config.yml:5]: The container spawned from this image will be the controller during integration test. So it needs drtools and your project code to work correctly. This avoids having to install drtools in a complicated way.

**Review by michcio1234 — APPROVED** [review]

**Pull request merged**